Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM buildpack-deps:stretch
+FROM fedora:latest
 
-MAINTAINER Sergei O. Udalov <sergei.udalov@gmail.com>
```

Comment:
=(

Reply:
Sorry, what's wrong? it is your image.

Reply:
I mean, why do we need to remove my name :)